src/harness/fourslash.ts

```diff
@@ -831,6 +831,7 @@ module FourSlash {
                 if (expectedText !== undefined) {
                     assert.notEqual(actualQuickInfoText, expectedText, this.messageAtLastKnownMarker("quick info text"));
                 }
+                // TODO: should be '==='?

```

Comment:
:+1: remove these comments. If these were causing issues, discuss in person with someone else on the team.